securedrop/models.py

```diff
@@ -286,7 +324,21 @@ def __init__(
     def __repr__(self) -> str:
         return f"<Reply {self.filename!r}>"
 
-    def to_json(self) -> "Dict[str, Any]":
+    def to_api_v2(self) -> Dict[str, Any]:
+        return {
+            "kind": "reply",
+            "uuid": self.uuid,
+            "source": self.source.uuid,
+            "size": self.size,
+            "journalist_username": self.journalist.username,
+            "journalist_first_name": self.journalist.first_name or "",
+            "journalist_last_name": self.journalist.last_name or "",
+            "journalist_uuid": self.journalist.uuid,
```

Comment:
By analogy from `.uuid` → `Source.uuid`, `.journalist_uuid` → `Journalist.uuid` should be all we provide here.

```suggestion
            "journalist_uuid": self.journalist.uuid,
```

Reply:
Done in eef1dceb6ea1bc2f18be06b3d806e430a3ec7a25.